README.md

```diff
@@ -1 +1,206 @@
+
+## Registratie van de applicatie
+
+Op de API store kan je uw eigen applicatie registreren. Deze dien je aan te maken in de organisatie waartoe de applicatie behoort.
+
+Indien er nog geen organisatie is dien je deze ook aan te maken.
+
+Voor iedere omgeving is er een aparte omgeving opgezet:
+* https://api-store-o.antwerpen.be
+* https://api-store-a.antwerpen.be
+* https://api-store.antwerpen.be
+
+Nadat je uw applicatie aangemaakt heb kan je een contract aanmaken met de A-Profiel en/of M-Profiel API.
+
+De OAuth2 client_id en client_secret kan je bij de detail gegevens van uw applicatie vinden. Afhankelijk van de API is er nog een goedkeuring nodig van de beheerder van de API. Deze goedkeuringsflow verloopt gehaal automatisch.
+
+De OAuth2 tokens zijn geldig voor alle OAuth2 services waarvoor uw applicatie een contract heeft.
+
+## Authenticeren als een applicatie
+
+### OAuth2 profiel: client_credentials
+
+![](img/client_credentials.png)
+
+#### 1) access_token ophalen
+
+*Request:*
+
+curl -X POST -d "client_id={{CLIENT_ID}}&client_secret={{CLIENT_SECRET}}&grant_type=client_credentials" https://api-gw-p.antwerpen.be/org/service/v1/oauth2/token
+
+*Response:*
+
+{"token_type":"bearer","access_token":"8c5b38ffee744ac6bd1e18eae5cbd406","expires_in":7200}
+
+#### 2) API oproepen
+*Request:*
+
+curl -v -H "Authorization: Bearer {{ACCESS_TOKEN_FROM_PREVIOUS_STEP}}" 'https://api-gw-p.antwerpen.be/astad/aprofiel/v1/me'
+
+## Authenticeren als een gebruiker
+
+### OAuth2 authorizatie server
+
+De OAuth2 authorize server kan je op volgende locatie vinden:
+* https://api-oauth2-o.antwerpen.be
+* https://api-oauth2-a.antwerpen.be
+* https://api-oauth2.antwerpen.be
+
+Een voorbeeld redirect naar de authorizatie ziet er zo uit:
+```
+https://api-oauth2.antwerpen.be/v1/authorize?
+response_type=code
+&service=astad.aprofiel.v1
+&client_id=YOUR_CLIENT_ID
+&scope=astad.aprofiel.v1.username%20astad.aprofiel.v1.name%20astad.aprofiel.v1.avatar%20astad.aprofiel.v1.email%20astad.aprofiel.v1.phone
+&redirect_uri=YOUR_REDIRECT_URI
+&state=thisParameterWillBeAddedToTheRedirectUri
+&save_consent=true
+```
+
+| Parameter | Verplicht | Omschrijving |
+| :---         |     :---:      |  :---   |
+| response_type   | true     | Voor de authorization_code flow dien je hier 'code' te gebruiken .    |
+| client_id     | true       | De client_id die je kan terugvinden bij uw applicatie in de API store.
+| scope     | true       | Een door comma's gescheiden lijst van scopes die de gebruiker dient goed te keuren.      |
+| redirect_uri     | true       | De redirect_uri die overeenkomt met diegene die is ingegeven in de API store voor uw applicatie.
+| redirect_uri_lng     | false       | Als deze waarde op 'true' staat zal de redirect_uri een extra parameter 'lng' hebben wanneer de gebruiker de OAuth2 authorize applicatie verlaat. De waarde voor de 'lng' querystring parameter zal overeenkomen met de taal op de Authorize applicatie. |
```

Comment:
**authorizatie** applicatie (2x)

Reply:
autorisatie

Reply:
Aangepast hoewel Authorize wel een bekend concept is binnen OAuth2.